src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -113,9 +140,19 @@ def fuzz(self, target_path, options, unused_reproducers_dir=None, max_time=0):
     self._create_temp_corpus_dir('new')
 
     args = options.arguments
-    args += ['--coverfile', runner.get_cover_file_path()]
+    args += [
+        '--coverfile',
+        runner.get_cover_file_path(), '--input', options.corpus_dir
+    ]
+
+    # Init corpus ahead of fuzzing
```

Comment:
These comments are not required, can be in the function description for those function.

Reply:
Done.